engine/src/main/java/org/terasology/rendering/nui/widgets/UIDropdown.java

```diff
@@ -264,31 +264,31 @@ public void changeHighlighted(boolean increase) {
 
     @Override
     public boolean onKeyEvent(NUIKeyEvent event) {
-        if (event.isDown() && TabbingManager.focusedWidget.equals(this)) {
-            if (opened) {
-                TabbingManager.setWidgetIsOpen(true);
-            } else {
-                TabbingManager.setWidgetIsOpen(false);
-            }
-
+        if (event.isDown()) {
             int keyId = event.getKey().getId();
-            if (keyId == Keyboard.KeyId.UP) {
-                this.changeHighlighted(false);
-                return true;
-            } else if (keyId == Keyboard.KeyId.DOWN) {
-                this.changeHighlighted(true);
-                return true;
-            }
-            if (keyId == Keyboard.KeyId.LEFT) {
-                if (opened) {
-                    setOpenedReverse(false);
-                }
-                return true;
-            } else if (keyId == Keyboard.KeyId.RIGHT) {
-                if (!opened) {
-                    setOpenedReverse(false);
-                }
-                return true;
+            switch (keyId){
```

Comment:
Another trivial missing space, but I'll leave any additional ones unremarked on as Checkstyle should catch them all 👍